assets/js/components/KeyMetrics/LostEventsSubtleNotification.js

```diff
@@ -47,8 +40,8 @@ export default function LostEventsSubtleNotification( {
 		)
 	);
 	return (
-		<SubtleNotification
-			className="googlesitekit-acr-subtle-notification googlesitekit-acr-subtle-notification__lost-events"
+		<Notice
+			type="warning"
```

Comment:
No need to hardcode string for type, we have an enum for that:
```suggestion
			type={ Notice.TYPES.WARNING }
```

This can be replaced in all other occurances across the files